core/src/main/java/cz/xtf/core/openshift/HelmBinary.java

```diff
@@ -0,0 +1,56 @@
+package cz.xtf.core.openshift;
```

Comment:
What about putting helm specific things into own package, e.g. cz.xtf.core.helm?

After removing dependency between HelmBinary and OpenShiftBinary there is no reason to be part of cz.xtf.core.openshift ?